### ISSUE TITLE
How should refunded DLCs be accounted for?

### ISSUE THREAD

**Issue by Christewart** [bitcoin-s/bitcoin-s#3261]
This is an on going debate between @benthecarman and I. In #3249 we modified the wallet level accounting logic to only account for `ClaimedDLCStatus` rather than `ClosedDLCStatus`. This means refunds (which do incur blockchain fees resulting in negative PNL!) are no longer accounted for on the wallet level.

Ben's argument against counting refunds is that it skews PNL. If I have 2 closed DLCs that 

1. Loses all of the collateral i contributed resulting in `-100%` rate of return
2. A refunded DLC that has a rate of return of `-1%` because of blockchain fees

that would result in a wallet level PNL of ~`-50%`. His argument is that this isn't useful which does hold some merit IMO.

**Comment by benthecarman** [comment]
Yes my opinion is that the DLC Accounting should take into account successful DLCs, I would not call a refunded DLC and it should be treated as if it never happened

**Comment by Christewart** [comment]
Related to #3235
